subsys/net/lib/tls_credentials/Kconfig

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig TLS_CREDENTIALS
```

Comment:
Btw, sooner or later someone will think "Is that subsys is for TLS credentials, or security credentials in general?". But I guess, it's too early for that.